src/toast/tests/ops_sim_ground.py

```diff
@@ -81,7 +81,7 @@ def test_exec(self):
                     "--site-alt",
                     "{}".format(site.earthloc.height.to_value(u.meter)),
                     "--patch",
-                    "small_patch,1,40,-40,44,-44",
+                    "tile000,1,40,0,60,-40",
```

Comment:
The unit test should stick to the `small_patch` target to keep the running time short.

Reply:
Ok, i will revert it now.